Qlc/Objects/RewardParameters.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Newtonsoft.Json;
+
+namespace Qlc.Objects
+{
+    public class RewardParameters
```

Comment:
change RewardParameters to MinerRewardParameter just for clear.

Reply:
Done